DevCycle.SDK.Server.Common/API/DVCBaseClient.cs

```diff
@@ -79,5 +79,18 @@ protected async Task<T> GetResponseAsync<T>(object body, string urlFragment, Dic
                 throw new DVCException(errorResponse);
             }
         }
+
+        public void validateEnvironmentKey(string environmentKey)
+        {
+            if (string.IsNullOrEmpty(environmentKey))
+            {
+                throw new ArgumentException("Missing environment key! Call build with a valid environment key");
+            }
+
+            if (!environmentKey.StartsWith("server") || !environmentKey.StartsWith("dvc_server"))
+            {
+                throw new ArgumentException("Invalid environment key provided. Please call build with a valid server environment key");
```

Comment:
Should explain what we're expecting too

Reply:
like tell the user that we're expecting something that starts with `server` or `dvc_server`?

Reply:
yeah something like that - or maybe update the docs too? https://docs.devcycle.com/docs/home/feature-management/organizing-your-flags-and-variables/api-and-sdk-keys

Reply:
ideally we'd just use the same message as NodeJS, or change the one there as well.

Reply:
I think it's better to just update the docs since that's usually the first place users will look on setup/when things go wrong